timing.py

```diff
@@ -75,7 +75,10 @@
                 mod.setup(Q, T)
 
                 elapsed_times = []
-                for _ in range(n_iter):
+                iter_count = 0
+                while sum(elapsed_times) < timeout and iter_count < iter_max:
```

Comment:
To avoid adding extra parameters, I'm reusing the existing `timeout` variable

Reply:
Sure! Though I feel like `timeout` means something different but it's not a big deal